t/cli.t

```diff
@@ -8,31 +8,35 @@ use File::Temp 'tempfile';
 
 use Capture::Tiny qw( capture capture_stdout );
 
+use experimental 'signatures';
+
 subtest help => sub {
-    my @commands = map s/^\s+|\s+$//gr, grep /^\s+/, split /\n/, capture_stdout {
+    my @commands = map s/^\s+|\s+$//gr, grep /^\s+/, split /\n/,
+        capture_stdout {
         system qw( bin/newrelic-admin help );
-    };
+        };
 
     ok @commands, 'Help lists available commands';
 
     subtest plain => sub {
-        like test('help'), qr/^Usage: newrelic-admin COMMAND/, 'Has usage line';
+        like test( ['help'] ), qr/^Usage: newrelic-admin COMMAND/,
+            'Has usage line';
     };
 
     subtest command => sub {
         for my $command (@commands) {
             subtest $command => sub {
-                like test( 'help', $command ),
-                    qr/^Usage: newrelic-admin (?:$command|COMMAND)/,
+                like test( [ 'help', $command ] ),
+                    qr/^Usage: newrelic-admin (?:$command|COMMAND)/,;
             };
         }
     };
 
     subtest option => sub {
         for my $command (@commands) {
             subtest $command => sub {
-                like test( $command, '--help' ),
-                    qr/^Usage: newrelic-admin (?:$command|COMMAND)/,
+                like test( [ $command, '--help' ] ),
+                    qr/^Usage: newrelic-admin (?:$command|COMMAND)/,;
```

Comment:
What is that `.;` at the end of the line? Looks wrong

Reply:
fixed. it looks like the test label was missing